tests/testndmapping.py

```diff
@@ -118,7 +118,8 @@ def test_idxmapping_redim(self):
         data = [((0, 0.5), 'a'), ((1, 0.5), 'b')]
         ndmap = MultiDimensionalMapping(data, kdims=[self.dim1, self.dim2])
         redimmed = ndmap.redim(intdim='Integer')
-        self.assertEqual(redimmed.kdims, [Dimension('Integer'), Dimension('floatdim')])
```

Comment:
I've implemented proper comparisons for lists so now it actually compares the dimensions using the comparison function defined on ``ComparisonTestCase``.